src/docs/templates/swell-template-edm.hbs

```diff
@@ -0,0 +1,886 @@
+        font-display: swap;
+        font-style: normal;
+        font-weight: 700;
+      }
+
+      @font-face {
+        font-family: "sofia-pro";
+        src: url("https://cdn.shopify.com/s/files/1/0286/1327/9779/t/122/assets/SofiaPro-Regular.woff2") format("woff2"),
+        url("https://cdn.shopify.com/s/files/1/0286/1327/9779/t/122/assets/SofiaPro-Regular.woff2") format("woff");
+        font-display: swap;
+        font-style: normal;
+        font-weight: 400;
+      }
+      
+      body {
+        margin: 0;
+        padding: 0;
+        -webkit-text-size-adjust: 100%;
+        -ms-text-size-adjust: 100%;
+        font-family: "sofia-pro";
+      }
+
+      img {
+        border: 0 none;
+        height: auto;
+        line-height: 100%;
+        text-decoration: none;
+        outline: none;
+        -ms-interpolation-mode: bicubic;
+      }
+
+      a img {
+        border: 0 none;
+      }
+
+      /* normalizes image behavior by removing padding on top and bottom in gmail / yahoo */
+      .image-reset img,
+      img.image-reset {
+        display: block;
+      }
+
+      table,
+      td {
+        border-collapse: collapse;
+        mso-table-lspace: 0pt;
+        mso-table-rspace: 0pt;
+      }
+
+      #wrapper-table {
+        margin: 0;
+        padding: 0;
+      }
+
+      /* fixes outlook appended ExternalClass to elements */
+
+      .ExternalClass {
+        width: 100%;
+      }
+
+      .ExternalClass,
+      .ExternalClass p,
+      .ExternalClass span,
+      .ExternalClass font,
+      .ExternalClass td,
+      .ExternalClass div {
+        line-height: 100%;
+      }
+
+      /* normalizes outlook generated view in browser link */
+
+      #outlook a {
+        padding: 0;
+      }
+
+      /* ========== end css reset ========== */
+
+      /* ========== utilities ========== */
+
+      .desktop-hide {
+        display: none;
+        padding: 0;
+        max-height: 0;
+        overflow: hidden;
+        float: left;
+        font-size: 0;
+        line-height: 0;
+        mso-hide: all;
+      }
+
+      .desktop-hide img,
+      img.desktop-hide {
+        max-height: 0;
+        width: 0;
+      }
+
+      .align-left {
+        text-align: left;
+      }
+      .align-center {
+        text-align: center;
+      }
+      .align-right {
+        text-align: right;
+      }
+
+      #navigation {
+        background-color: #ffffff;
+        border-top: 1px solid #cacaca;
+        border-left: 1px solid #cacaca;
+        border-bottom: 1px solid #cacaca;
+        border-right: 1px solid #cacaca;
+      }
+
+      #navigation td {
+        text-align: center;
+      }
+
+      #navigation td a {
+        color: #606060;
+        font-family: Arial, Helvetica, sans-serif;
+        font-style: italic;
+        font-size: 16px;
+        text-decoration: none;
+      }
+
+      #navigation .nav-spacer {
+        background-color: #cacaca;
+      }
+
+      /* ========== end utilities ========== */
```

Comment:
if we are not using these classes let's delete them, same for other, the important are only the rules of reset and grid